Command/IndexImportCommand.php

```diff
@@ -53,13 +59,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $manager = $this->getManager($input->getOption('manager'));
 
+        $options = [];
+        if ($input->getOption('gzip')) {
+            $options['gzip'] = true;
+        }
+
         /** @var ImportService $importService */
         $importService = $this->getContainer()->get('es.import');
         $importService->importIndex(
             $manager,
             $input->getArgument('filename'),
             $output,
-            $input->getOption('bulk-size')
+            $input->getOption('bulk-size'),
```

Comment:
Actually, it must be in `$options`, that's why this variable was introduced, to collect all options in a single place.